### ISSUE TITLE
 An unhandled exception has occurred while executing the request.

### ISSUE THREAD

**Issue by hujiyi** [liangshiw/Lsw.Abp.AspNetCore.Mvc.UI.Theme.Stisla#8]
```csharp
    [DependsOn(
        typeof(BookStoreHttpApiModule),
        typeof(BookStoreApplicationModule),
        typeof(BookStoreEntityFrameworkCoreDbMigrationsModule),
        typeof(AbpAutofacModule),
        typeof(AbpIdentityWebModule),
        typeof(AbpAccountWebIdentityServerModule),
        typeof(AbpAspNetCoreMvcUiStislaThemeModule),       
        // typeof(AbpAspNetCoreMvcUiBasicThemeModule),
        typeof(AbpAspNetCoreAuthenticationJwtBearerModule),
        typeof(AbpTenantManagementWebModule),
        typeof(AbpAspNetCoreSerilogModule)
        )]
    public class BookStoreWebModule : AbpModule
```
publish the project , and run the command in the powershell:
```bash
  dotnet .\Acme.BookStore.Web.dll
```
throwing an exception：
```bash
[09:43:13 ERR] An unhandled exception has occurred while executing the request.
Volo.Abp.Minify.NUglify.NUglifyException: There are some errors on uglifying the given source code!
__bundles/Stisla.Global.C82104EA56ED2F79F6358CDB50B0BA77.css(2790,24): run-time error CSS1019: Unexpected token, found '}'
__bundles/Stisla.Global.C82104EA56ED2F79F6358CDB50B0BA77.css(2797,1): run-time error CSS1019: Unexpected token, found '@-webkit-keyframes'
__bundles/Stisla.Global.C82104EA56ED2F79F6358CDB50B0BA77.css(2798,3): run-time error CSS1062: Expected semicolon or closing curly-brace, found '0%'
   at Volo.Abp.Minify.NUglify.NUglifyMinifierBase.CheckErrors(UglifyResult result)
   at Volo.Abp.Minify.NUglify.NUglifyMinifierBase.Minify(String source, String fileName)
   at Volo.Abp.AspNetCore.Mvc.UI.Bundling.BundlerBase.AddFileToBundle(IBundlerContext context, StringBuilder bundleContentBuilder, String fileName)
```
Using the original UI `typeof(AbpAspNetCoreMvcUiBasicThemeModule),`  is no problem

**Comment by realLiangshiwei** [comment]
I will check this

**Comment by realLiangshiwei** [comment]
@hujiyi 
Fixed by : https://github.com/liangshiw/Lsw.Abp.AspNetCore.Mvc.UI.Theme.Stisla/commit/7adf683ce8d9bd404236f504a114266dd17a1f01
Update version to 1.3.1

**Comment by hujiyi** [comment]
> 
> 
> @hujiyi
> Fixed by : [7adf683](https://github.com/liangshiw/Lsw.Abp.AspNetCore.Mvc.UI.Theme.Stisla/commit/7adf683ce8d9bd404236f504a114266dd17a1f01)
> Update version to 1.3.1

ths
